cli/index.js

```diff
@@ -54,17 +53,16 @@ const outputMode = cli.flags.output || 'pretty';
 const outputPath = cli.flags.outputPath || 'stdout';
 const flags = cli.flags;
 
-const driver = new ChromeProtocol();
-
 // set logging preferences
+flags.logLevel = 'info';
```

Comment:
The module defaults to quiet; the CLI defaults to info and sets that in the module.